spec/features/process_a_request_accept_spec.rb

```diff
@@ -6,26 +6,81 @@
 
   include_context 'process request setup'
 
-  around do |ex|
-    travel_to(Date.new(2016, 12, 1)) { ex.run }
+  context 'with visitor contact list', vcr: { cassette_name: 'process_happy_path_with_contact_list' } do
+    # Leicester has contact list enabled in 'test'
```

Comment:
I would have preferred to keep the test setup in the test file and avoid:
```ruby
config.staff_prisons_with_nomis_contact_list = ['Leicester']
```
under config/envornments/test.rb